launcher/vue.config.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-unused-vars
 const path = require("path");
```

Comment:
can we remove this?

Reply:
not sure, @MaxTheGeeek what do you think? probably need somewhere in frontend?

Reply:
tested, and `import-config` works fine without this. So I just remove it for now. :)

Reply:
> can we remove this?

I'd remove it a few times but thought maybe someone needs

Reply:
acutally i ment this:  `// eslint-disable-next-line no-unused-vars` 😅

Reply:
done!

Reply:
> acutally i ment this: `// eslint-disable-next-line no-unused-vars` 😅

That belonged to the path also.